src/appleseed/renderer/modeling/project/projectfilereader.cpp

```diff
@@ -3036,15 +3038,49 @@ namespace
     }
 }
 
+bool is_packed_project(const string& project_filepath, string& project_name) {
+    try 
+    {
+        vector<string> appleseed_files = get_filenames_with_extension_from_zip(project_filepath, ".appleseed");
+        
+        if (appleseed_files.size() != 1)
+            throw ExceptionUnsupportedFileFormat("Should be exactly one .appleseed file in project");
```

Comment:
You're right, I had overlooked that.

Still, a `is_something()` method shouldn't throw an exception. Maybe inline this whole function where it's used instead, and get rid of the function?